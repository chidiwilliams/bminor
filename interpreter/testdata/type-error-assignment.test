b: array [2] boolean = {true,false};
x: integer = 0;
x = b[0];      // error: x is not a boolean!
