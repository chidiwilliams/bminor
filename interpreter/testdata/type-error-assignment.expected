err: Error: x is not a(n) boolean
